meeting-notes/2018/2018-08-20--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,204 @@
+     - rmdir/del -> unpin; for files directly under the ipfs root
+     - write -> add -w
+ 
+@diasdavid (Unsure if will have good connectivity for this one)
+ - Done:
+   - Done more interviews (almost there, only 3 that still need to be scheduled)
+   - Request DX Team to get go-ipfs, go-libp2p and go-ipld Packages table https://github.com/ipfs/testing/issues/128
+ - Blocked:
+ - Next: 
+   - Finish the interviews/chats
+   - Compile all the notes from the interviews/chats into something easy to consume by everyone in the Go Core Dev Teams
+   
+@kevina
+  - Done:
+    - In Progress: Begin new "cid" sub-command https://github.com/ipfs/go-ipfs/pull/5385
+    - Needs Review: Extract parts of go-cid into go-cidutil https://github.com/ipfs/go-cid/pull/69 
+    - In Progress: Add support for inlinling via the id-hash https://github.com/ipfs/go-ipfs/pull/5281
+    - [WIP] Start work towards switching blockstore to use Multihashes: https://github.com/ipfs/go-ipfs-blockstore/pull/13
+    - Various work towards Base32 CidV1 (see Other notes section)
+  - Blocked:
+     - Various aspects of Base32 CidV1 need feedback (see Other notes section)
+  - Next:
+     - Continue Base32 CidV1 work
+     - Continue In progress P.R.
+
+@schomatis (not attending)
+ - Done:
+   - `gx`: 
+     - Unify dependency fetching in `gx` (https://github.com/whyrusleeping/gx/pull/197)
+     - Review part of the documentation (https://github.com/whyrusleeping/gx/issues?q=is%3Aissue+is%3Aopen+label%3Adocumentation).
+   - Encourage some community contribution.
+   - In progress: Badger integrity (`go-ipfs`: #5213, #5280) and memory (#5177, #5390) issues.
+   	- - Notes from @bigs:
+     	 - When Commit(nil) returns (as we have it implemeneted in go-ds-badger), entries have been committed to the WAL at least and should be truncate-safe.
+ - Next:
+   - Ease the use of `gx-go link` when dependencies don't match, e.g., developing in `go-unixfs` and testing in `go-ipfs` (https://github.com/whyrusleeping/gx-go/issues/46).
+   - Keep working on `gx` docs and code clarity.
+   - Incorporate the feedback of the DAG traversal PR (https://github.com/ipfs/go-ipfs/pull/5257) and split it in the corresponding repos (`go-unixfs` and `go-ipld-format`). PR under review.
+
+@stebalien
+ - Done:
+   - IPLD:
+     - thinking through go-ipld interfaces from a performance perspective (no concrete progress)
+     - have a plan to fix the interfaces and move to the IPLD org: https://github.com/ipld/go-ipld/issues/2
+   - fix (in review): stop go-libp2p from NAT port mapping non-nat (e.g., localhost) addresses.
+   - fix (in review): stop go-ipfs from listening on random ports.
+   - misc fixes, reviews, gx
+ - Next:
+   - multistream-select (again)
+   - continue work on IPLD interfaces/migration.
+ - Blocked:
+   - @daviddias: [Remove bitswap unwant](https://github.com/ipfs/go-ipfs/pull/5308)
+   - @daviddias: [New multiaddr protocol](https://github.com/multiformats/multiaddr/pull/68)
```

Comment:
I've followed up. Feel that I need to do lot more reading before signing off such a thing (lot's of RFCs to read and context to absorb) and unfortunately I've very limited time right now to go that deep exploring.

Can you give me a date in which this becomes critical to not be solved? Or, better, can do you a 5 minute presentation with your brain dump?